rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -601,13 +601,9 @@ rocalResizeMirrorNormalize(
         output->reset_image_roi();
 
         std::shared_ptr<ResizeMirrorNormalizeNode> rmn_node =  context->master_graph->add_node<ResizeMirrorNormalizeNode>({input}, {output});
-        // RPP doesn't support returning float buffers so passing 0 and 1 as mean and std and doing normalization in rocAL
-        // TODO: To be removed with rocAL Tensor support
-        // rmn_node->init(0, 1, mirror);
         rmn_node->init(mean, std_dev, mirror);
-        // TODO: Uncomment the below lines once RMN meta node is added to ToT
-        // if (context->master_graph->meta_data_graph())
-        //     context->master_graph->meta_add_node<ResizeMirrorNormalizeMetaNode,ResizeMirrorNormalizeNode>(rmn_node);
+        if (context->master_graph->meta_data_graph())
```

Comment:
where is the init called here?

Reply:
meta_data_graph is created in this function in master_graph.cpp - create_coco_meta_data_reader